cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+# Table of contents: 
+[Dynamic Objects (Only Supported in JS Objects)](#dynamic-objects-only-supported-in-js-objects)
+ - [Supported Dynamic Glue Codes](#supported-dynamic-glue-codes)
+   - [Background](#background)
+   - [Firebolt object](#firebolt-object)
+     - [Sample Firebolt Object Format](#sample-firebolt-object-format)
+   - [Runtime Variables](#runtime-variables)
+   - [ResolveAtRuntime Function](#resolveatruntime-function)
+     - [Examples](#examples)
+       - [Example 1](#example-1)
+       - [Example 2](#example-2)
+       - [Example 3](#example-3)
+   - [Usage](#usage)
+     - [Example 1](#example-1-1)
+     - [Example 2](#example-2-1)
+   - [Advanced Support](#advanced-support)
+     - [Examples](#example-3)
+
+# Dynamic Objects (Only Supported in JS Objects)
+
+## Supported Dynamic Glue Codes
+To use dynamic firrebolt objects, we need to use dynamic glue codes listed [here](../../support/step_definitions/dynamicCalls.md)
+
+- we test the '(.+)' getters and setters(?: '(._?)'(?: to '(._?)')?)?
+- 1st party app invokes the '(.+)' API to set( invalid)? value
+- '(.+)' invokes the '(.+)' get API
+- '(.+)' registers for the '(.+)' event
+- '(.+)' platform responds to '(.+)' (get|set) API(?: with '(.+)')?
+- '(.+)' platform (triggers|does not trigger) '(.\*?)' event(?: with '(.+)')?
+
+## Background
+
+FCS supports dynamic firebolt JS objects to be used across multiple examples in a scenario outline. This is applicable for test cases where we can validate setter response, getter response, event response etc using a single JS object. More details outline below.
+
+## Firebolt object
+
+- Firebolt objects can be added in Javascript files located in the `cypress/fixtures/fireboltCalls` folder. Ex: `cypress/fixtures/fireboltCalls/accessibility.js`
+- Firebolt objects present in the config module will take priority if the same key present in FCS.
+
+### Sample Firebolt Object Format
+
+Below is the sample format showing how to create a firebolt object.
+
+```
+FIREBOLT_CALL = {
+    method: ...,
+    params: ...,
+    validationJsonPath: ...,
+    setMethod: ...,
+    setParams: ...,
+    setValidationJsonPath: ...,
+    setContent: ...,
+    event: ...,
+    eventValidationJsonPath: ...,
+    content: ...
+  };
+```
+
+| **Param**               | **Definition**                                                                                            | **Example**                                   | **Supported Type**             |**Required** |
+| ----------------------- | --------------------------------------------------------------------------------------------------------- | --------------------------------------------- | ------------------------------ | --------------|
```

Comment:
Just give N in Required column since no field is mandatory

Reply:
removed required column